Backend/app/user/user_service.py

```diff
@@ -201,6 +209,55 @@ def upgrade_to_regular_licence(
         except sqlite3.IntegrityError:
             return {"error": "No se pudo actualizar el usuario"}
 
+    def upgrade_role_to_librarian(
```

Comment:
Please mark this with a TODO to move this to a librarian service file later. Right now there's a lot of people working in parallel so it'd be more trouble than it's worth but user_service is already bloated enough.